### PR TITLE
Update Changelog

### PR DESCRIPTION
**Pull request by woop**
**What this PR does / why we need it**:
It updates the current change log up to Feast 0.4.3

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by woop** [comment]
Ran into the following error during changelog generation

```
Warning: PR 412 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 407 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 388 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 382 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 380 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 374 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 372 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 370 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 360 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 244 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 241 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 240 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 238 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 236 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 235 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 234 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 232 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
Warning: PR 227 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
```

This means these PRs were NOT considered. Could be due to the PRs only being found on the non-master branch. Please go through them manually to ensure that they are captured.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,132 @@
 # Changelog
 
+## [v0.4.3](https://github.com/gojek/feast/tree/v0.4.3) (2020-01-08)
+
+[Full Changelog](https://github.com/gojek/feast/compare/v0.4.2...v0.4.3)
```
Comment (zhilingc) [inline CHANGELOG.md]: 0.4.3 Changes:
- Bugfix for redis ingestion retries throwing NullPointerException on remote runners

**Comment by khorshuheng** [comment]
> Ran into the following error during changelog generation
> 
> ```
> Warning: PR 412 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 407 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 388 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 382 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 380 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 374 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 372 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 370 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 360 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 244 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 241 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 240 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 238 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 236 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 235 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 234 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 232 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> Warning: PR 227 merge commit was not found in the release branch or tagged git history and no rebased SHA comment was found
> ```
> 
> This means these PRs were NOT considered. Could be due to the PRs only being found on the non-master branch. Please go through them manually to ensure that they are captured.

- PR 244 and older are merged on 0.3-dev (or older), and was already on 0.3.0. Those can be ignored.
- PR 412, 407, 360 are merged in branch 0.3 but not released.
- PR 370, 372, 374, 388 are merged into project namespacing branch.

File: CHANGELOG.md (hunk 1 above)
Comment (khorshuheng) [reply CHANGELOG.md]: Updated

**Pull request merged**